### PR TITLE
⚡️ Release 3.0.0

### PR DESCRIPTION
**Pull request by dplewis**
A lot of new additions and a breaking change in this one. First release of the New Year

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
 
 ### master
-[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.19.0...master)
+[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/3.0.0...master)
+
+## 3.0.0
+[Full Changelog](https://github.com/parse-community/Parse-SDK-JS/compare/2.19.0...3.0.0)
+
+**Breaking Changes**
+For security purposes, logIn will default to `POST` instead of `GET` method.
```
Comment (mtrezza) [inline CHANGELOG.md:10]: Is there a PR reference for this or has this been a sec adv?

Comment (dplewis) [reply CHANGELOG.md:10]: I'll add it

Comment (mtrezza) [reply CHANGELOG.md]: I like that the changelog explains the breaking change in some detail. We should aim for that in any changelog.

**Review by mtrezza — APPROVED** [review]
LGTM!

**Pull request merged**